models/turbine_models/custom_models/sd_inference/clip.py

```diff
@@ -6,6 +6,7 @@
 
 import os
 import sys
+import re
```

Comment:
nit: looks unused?

Reply:
line 166

Reply:
(before my patch this file errored on undefined re)